src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/controller/VisitController.kt

```diff
@@ -335,6 +336,18 @@ class VisitController(
       example = "2021-11-03T09:00:00",
     )
     endDateTime: LocalDateTime?,
+    @RequestParam(value = "sessionTemplateReference", required = false)
+    @Parameter(
+      description = "Filter results by session template",
+      example = "v9-d7-ed-7u",
+    )
+    sessionTemplateReference: String?,
```

Comment:
should these be moved to a different endpoint instead?

Reply:
I would say so because if you have the reference nothing else counts